File: Python/Sunny_Number.py
print("Sunny Number Checker\n")
n = int(input("Enter the number : "))
if (n+1) ** 0.5 == int((n+1) ** 0.5):
	print(f"{n} is an sunny number.")
else:
	print(f"{n} is not an sunny number.")

File: Java/src/Sunny_Number.java
/**
 * Sunny Number Checker
 * 
 * @author Jaysmito Mukherjee 
 * @version 2019-A-025
 */
import java.util.*;
public class Sunny_Number
{
    public static void main()
    {
        System.out.print("\f");
        Scanner sc=new Scanner(System.in);
        System.out.print("Sunny Number Checker\n~~~~~ ~~~~~~ ~~~~~~~\nEnter the number: ");
        int n=sc.nextInt();
        if((int)Math.sqrt(n+1)==(int)Math.ceil(Math.sqrt(n)))
            System.out.print(n+" is a Sunny number.");
        else
            System.out.print(n+" is not a Sunny number.");        
    }
}
